cli/cmd/add/component-version/cmd.go

```diff
@@ -290,7 +290,7 @@ func AddComponentVersion(cmd *cobra.Command, _ []string) error {
 	}
 	defer stop()
 
-	_, err = constructor.ConstructDefault(cmd.Context(), constructorSpec, opts)
+	_, _, err = constructor.ConstructDefault(cmd.Context(), constructorSpec, opts)
```

Comment:
will be used in https://github.com/open-component-model/open-component-model/pull/1076